### ISSUE TITLE
At least notify user when a new update is available. 

### ISSUE THREAD

**Issue by pixincreate** [johanw666/Signal-Android#54]
- [x] I have searched open and closed issues for duplicates
- [ ] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Feature description
Notify user when a new update is available.     
This can done by fetching this repo's latest release. Just compare whether the current version is lesser than or equal to the ones in the release channel, if yes, just show a pop-up at the top with the link. Clicking on it redirects the user to the release page. 

#
### Device info

**Device:** DRG
**Android version:** 12.1
**Signal version:** 5.42.1

**Comment by soopstar** [comment]
I typically just watch the repo, so when a new update is released, I get an email with the link and whatnot. Of course, an in-app update checker would be nice, too.

**Comment by xKippi** [comment]
You should see that an update is available in the app store you are using. I don't know for sure if this applies to the Google Play Store, but at least in Aurora Store it is like this.

**Comment by pixincreate** [comment]
> You should see that an update is available in the app store you are using. I don't know for sure if this applies to the Google Play Store, but at least in Aurora Store it is like this.

This is a fork of Signal and not the Official Signal itself. It has different signatures and doesn't match with the one that is on the Google Play. JW versions are different ad has many features compared to the official and has to be installed from the releases section of this repo.

**Comment by xKippi** [comment]
>  It has different signatures and doesn't match with the one that is on the Google Play

Yeah but it has the same package name, so although you can't install it, it will still show that a new version is available. And as i said I don't know the exact behavior from the Google Play store since I am using Aurora Store. There it shows me when an update is available. And of course I can't install it from there, I still have to get it from GitHub but at least I get notified. This is exactly what you asked for.

**Comment by johanw666** [comment]
You can set a Github notification that you are warned when a new release of a repository is published. Those notifications come by email, and in the Github Android app if you use it.

**Comment by pixincreate** [comment]
> You can set a Github notification that you are warned when a new release of a repository is published. Those notifications come by email, and in the Github Android app if you use it.

This is what I'm currently using and it's really annoying and not feasible in the long run!  
I'd really appreciate a built-in update checker the fetches latest version from `releases/latest` from GitHub and compare it with the current version of the app. FYI, even Molly.im has a built-in updater in case if you change your mind and is willing to add it here.

**Comment by karano-techie** [comment]
If you are using an rss reader  you can easily add update feeds for the releases of any github repo.
For me its been convenient way currently. 
Also i wouldn't like the signal app to give me a pop-up every week to remind me of an update.
url for adding to rss feed https://github.com/johanw666/Signal-Android/releases.atom

**Comment by blaugrau3001** [comment]
You can also use something like [Obtainium](https://github.com/ImranR98/Obtainium)
